src/main/java/org/eclipse/tractusx/ssi/lib/did/web/DidWebResolver.java

```diff
@@ -84,6 +84,7 @@ public DidDocument resolve(Did did) throws DidResolverException {
 
       return new DidDocument(json);
     } catch (Exception e) {
+      Thread.currentThread().interrupt();
```

Comment:
We should only interrupt the thread when we have InterruptedException in the catch.
Here we have a generic Exception, can we split the catch block?